docs/source/installed_packages.rst

```diff
@@ -0,0 +1,44 @@
+Using Installed Packages
+========================
+
+
+Making PEP 561 compatible packages
+**********************************
+
+Packages that supply type information should put a ``py.typed`` in their package
+directory. For example, with a directory structure as follows:
+
+.. code-block:: text
+
+setup.py
+package_a/
+    __init__.py
+    lib.py
+    py.typed
+
+the setup.py might look like:
+
+.. code-block:: python
+
+from distutils.core import setup
+
+setup(
+    name="SuperPackage",
+    author="Me",
+    version="0.1",
+    package_data={"package_a": ["py.typed"]},
+    packages=["package_a"]
+)
+
+
+Using PEP 561 compatible packages with mypy
+*******************************************
+
+PEP 561 specifies a format to indicate a package installed in site-packages or
+dist-packages supports providing type information. Generally, you do not need
+to do anything to use these packages. They should be automatically picked up by
+mypy and used for type checking.
+
+If you use mypy to type check a Python other than the version running mypy, you
+can use the ``--python`` flag to point to the executable, and mypy will pick up
+the site/dist-packages for the Python executable pointed to.
```

Comment:
Fair point, technically it searches eg `C:\Python36\` as well on Windows. I'm not sure there is a name that has a special meaning for the directories I mean beyond "installed package directories" which is a little vague. Perhaps this is clearer and enough?
`... mypy will find packages installed for that python executable`. I don't think it is important for the person to know *where* these directories are, as that is an abstraction made by pip/the OS. It is not at all standard, but it is almost always site- or dist-packages.

Reply:
Maybe refer to it as "the site packages directory"? I betcha people who aren't aware of all the details still know about site-packages and talk about it like that. Using two words instead of hyphenated "site-packages" is a subtle hint that this is about the general mechanism and not the specific directory.